packages/dx-react-grid/docs/reference/toolbar.md

```diff
@@ -0,0 +1,48 @@
+# Toolbar Plugin Reference
+
+A plugin that renders Grid's toolbar.
+
+## User Reference
+
+### Dependencies
+
+none
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+rootComponent | ElementType&lt;[ToolbarRootProps](#toolbarrootprops)&gt; | | A component that renders the toolbar root element.
+
+## Interfaces
+
+### ToolbarRootProps
+
+Describes properties passed to a component that renders the toolbar root element.
+
+Field | Type | Description
+------|------|------------
+children? | ReactElement | A React element to be placed in the toolbar.
+style? | Object | Styles that should be applied to the root toolbar element.
```

Comment:
Is the style property is needed here?

Reply:
Yes, there are styles that should be applied (BS3). I'm not sure we need to add the note about BS3.

Reply:
Are you sure that it is really needed to be documented? For me, it seems redundant.